index.js

```diff
@@ -0,0 +1,32 @@
+import {
+  createConnection,
+  ProposedFeatures,
+  TextDocuments
+} from 'vscode-languageserver/node.js'
+import {TextDocument} from 'vscode-languageserver-textdocument'
+import {configureUnifiedLanguageServer} from './server.js'
+
+/**
+ * @typedef {Pick<
+ *   import('unified-engine').Options,
+ *   'ignoreName' | 'packageField' | 'pluginPrefix' | 'plugins' | 'rcName'
+ * > & {
+ *   defaultSource?: string
+ * }} UnifiedLanguageServerOptions
```

Comment:
```suggestion
 * }} Options
```

I think namespacing types isn’t needed. `import type {Options} from 'unified-language-server'` is explicit enough?

Reply:
I personally like more explicit names, but it’s just a personal preference. I changed it to your suggestion.